android.pri

```diff
@@ -1,9 +1,26 @@
 include($$PWD/libs/qtandroidserialport/src/qtandroidserialport.pri)
 message("Adding Serial Java Classes")
 QT += androidextras
+
 ANDROID_PACKAGE_SOURCE_DIR = $$PWD/android
+exists($$PWD/custom/android) {
+    message("Merging $$PWD/custom/android/ -> $$PWD/android/")
+    ANDROID_PACKAGE_SOURCE_DIR = $$OUT_PWD/ANDROID_PACKAGE_SOURCE_DIR
+
+    system($$QMAKE_MKDIR $$ANDROID_PACKAGE_SOURCE_DIR)
+    system($$QMAKE_COPY_DIR $$PWD/android/* $$OUT_PWD/ANDROID_PACKAGE_SOURCE_DIR)
+    system($$QMAKE_COPY_DIR $$PWD/custom/android/* $$OUT_PWD/ANDROID_PACKAGE_SOURCE_DIR)
```

Comment:
`androiddeployqt` [only allows](https://doc.qt.io/qt-5/deployment-android.html#androiddeployqt) a single input folder so i had no better ideas but to merge these two folders in the build tree and give that to `androiddeployqt`